lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -87,9 +92,13 @@ def __init__(self, sgi_if_name: str, s1_if_name: str, gw_ip: str, enabled=True,
         self.sgi_if_name = sgi_if_name
         self.s1_if_name = s1_if_name
         self.ul_src_mac = self._get_mac_address(sgi_if_name)
-        self.ul_gw_mac = self._get_mac_address_of_ip(gw_ip)
         self.sgi_if_index = self._get_ifindex(self.sgi_if_name)
-        self.enabled = enabled
+        self.ul_gw_mac = self._get_mac_address_of_ip(gw_ip)
+
+        if self.ul_gw_mac is None:
```

Comment:
Added this because of the `IP: 2020::1` edge case

Reply:
what do you mean?

Reply:
When running integ test there seems to be an ipv6 ip passed which produces no mac lookup

Reply:
I see, lets check for IPv4 GW address and then only initialize it the ebpf manager object.

Reply:
I can fail the check if its not ipv4 but the current code can handle IPv6 already, it just handles an edge case where the passed IP address can't be looked up and matched to a mac. Lmk if you think we should drop the ipv6 support completely for now though